docs/README.md

```diff
@@ -0,0 +1,17 @@
+# Documentation
+
+Welcome to our documentation pages! What would you like to view?
```

Comment:
what about adding a short description about ESLint on the top?

Reply:
@Aladdin-ADD Thanks for the suggestion! I feel that would be slightly redundant given the homepage and About pages. If you feel strongly, you or I could always add that in another PR. I'm going to go ahead and merge as is since this is easy to correct and won't impact users of ESLint (the library).